tools/windows-node-installer/cmd/azure.go

```diff
@@ -65,19 +85,19 @@ func azCreateCmd() *cobra.Command {
 			"The created instance would be used as a worker node for the OpenShift Cluster.",
 		TraverseChildren: true,
 		RunE: func(_ *cobra.Command, _ []string) error {
-			os.Setenv("AZURE_AUTH_LOCATION", azCreateFlagInfo.credentialPath)
-			getFileSettings, err := auth.GetSettingsFromFile()
-			if err != nil {
-				return fmt.Errorf("unable to get the settings from path: %v", err)
-			}
-			if azCreateFlagInfo.credentialAccountID == "" {
-				azCreateFlagInfo.credentialAccountID = getFileSettings.GetSubscriptionID()
+			if azCreateFlagInfo.subscriptionID == "" {
+				subscriptionID, err := setEnvVariable(azCreateFlagInfo.credentialPath)
+				azCreateFlagInfo.subscriptionID = subscriptionID
+				if err != nil {
+					return fmt.Errorf("eror getting subscription ID, %v", err)
+				}
```

Comment:
will update createCmd to azCreateCmd, thought I looked at this, but clearly I messed up.

Reply:
fixed that in the comment section of the code.